mapcomposer/buildjs.cfg

```diff
@@ -234,7 +234,6 @@ include =
     plugins/WMSGetFeatureInfoMenu.js
     plugins/Navigation.js
     plugins/LayerTree.js
-    plugins/AddLayers.js
```

Comment:
This is the main AddLayers plugin, the default MapStore will not load without this.